### ISSUE TITLE
我觉得你这个颜色按钮位置有问题. 我找了一天都不知道在哪里

### ISSUE THREAD

**Issue by ByYogi** [BdR76/CSVLint#80]
The default color highlighting is foreground color. 
I'd like to change it to background color, but I can't find a way to do it. 
I searched for about 3 hours, fanally I found what I was looking for in this [document](https://github.com/BdR76/CSVLint/tree/master/docs#settings)

So, I think putting the button in this position is inhuman.

* The help me document: 

![image](https://github.com/BdR76/CSVLint/assets/111735598/d011cc6c-6c31-47a0-a2a5-d73545edcd51)

**Comment by BdR76** [comment]
I have updated the documentation, and moved the syntax highlighting colors to the top. Syntax highlighting is an important feature and shouldn't be all the way at the bottom of the documentation.

**Comment by ByYogi** [comment]
> Thanks for posting your issue, but I'm not sure what I can do about this. I try to make it as user-friendly as possible, so I'm sorry to hear you couldn't find this option right away. Where did you expect this option to be, where were you looking for it?
> 
> The colors can be changed in the settings dialog, imho that is the logical place. I could make the button a bit bigger or add the colored "Aa" icon to it, so that the button stands out a bit more on the dialog, if that's what you mean?

Thanks for noticing.

* I prefer to look for the configuration I want in the following places
![image](https://github.com/BdR76/CSVLint/assets/111735598/d441cc64-329b-48d0-8cc2-1bedf87946a2)
![image](https://github.com/BdR76/CSVLint/assets/111735598/3a4da0a8-f47d-4f7a-80c3-864ff5578ab8)

* Or you can change the description of the button to "Highlight". It's easier to understand the meaning than "Color".
![image](https://github.com/BdR76/CSVLint/assets/111735598/9996d10c-b62f-4ab3-ac78-808b1e1c24d8)

**Comment by ByYogi** [comment]
I think the biggest problem is, that this button is not reminiscent of a button with options. 
It's mistaken for a click function, like the ok and cancel buttons in the bottom right corner.

I think the simplest change would be to change the description info.
`Highlight ▾`

![image](https://github.com/BdR76/CSVLint/assets/111735598/43760fcb-f975-42ff-8ce2-bde8fd71c0ac)

**Comment by BdR76** [comment]
I've updated the color button using [this example](https://stackoverflow.com/a/27173509/1745616). I also tried adding the color icon to the button (on the right), but I think it looks better without the icon (on the left).

![csv_menu_button](https://github.com/BdR76/CSVLint/assets/7793956/62b767b0-bef9-4850-b9da-63acaaf29f51)

Are you able to download the ["beta" build DLL](https://github.com/BdR76/CSVLint/tree/master/CSVLintNppPlugin/bin) to check if the button displays correctly also on your system? It will be in the next release

**Comment by ByYogi** [comment]
On my win10 system it doesn't look displaying correctly
![image](https://github.com/BdR76/CSVLint/assets/111735598/bad611ce-c952-4d60-a320-abffa04a90cf)

**Comment by ByYogi** [comment]
Here's a dll I tried to make from the source code.
![image](https://github.com/BdR76/CSVLint/assets/111735598/7e489420-fda8-4c19-a458-f0ef7e69947c)

[CSVLint.zip](https://github.com/BdR76/CSVLint/files/13628423/CSVLint.zip)

**Comment by BdR76** [comment]
Thanks for posting the screenshots, this was really helpful. It looks like the Highlighting-button was not wide enough. It was 100px and I've updated it to be 128px, also see the ["beta" build DLL](https://github.com/BdR76/CSVLint/tree/master/CSVLintNppPlugin/bin).

I saw that you changed the button to also display the icon, and show the selected icon as well. However, I don't think this could work generally, because when you've just started the Settings dialog and haven't selected or changed anything, then the plugin cannot know which of the 4 items was selected previously.

This is due to the fact that only the 12 colors are stored, and the user can change these colors to anything they like in the preferences dialog. In other words, the selected menu item (1 through 4) is not stored anywhere.
